python/cudf/cudf/core/column/datetime.py

```diff
@@ -324,17 +324,8 @@ def to_pandas(
         # `copy=True` workaround until following issue is fixed:
         # https://issues.apache.org/jira/browse/ARROW-9772
 
-        if PANDAS_GE_200:
-            host_values = self.to_arrow()
-        else:
-            # Pandas<2.0 supports only `datetime64[ns]`, hence the cast.
-            host_values = self.astype("datetime64[ns]").to_arrow()
-
-        # Pandas only supports `datetime64[ns]` dtype
-        # and conversion to this type is necessary to make
-        # arrow to pandas conversion happen for large values.
         return pd.Series(
-            host_values,
+            self.to_arrow(),
```

Comment:
Just in noting in the future we should pass pandas a numpy array instead of a pyarrow array and `pandas.Series` treats pyarrow arrays as objects